doc/source/user_guide/getting_help.rst

```diff
@@ -28,6 +28,24 @@ The ``quick search`` field located in the navigation bar of the html
 documentation can be used to search for specific keywords (segmentation,
 rescaling, denoising, etc.).
 
+API Discovery
+-------------
+
+The users can make use of ``np.lookfor`` method to check the functions 
```

Comment:
Yes this paragraph can be improved. A loose suggestion:

```
NumPy provides a ``lookfor`` function to search APIs. By default ``lookfor`` will search the NumPy API:

> NumPy lookfor example, e.g. np.lookfor('eigenvector')

But it can also be used to search in modules, by passing either the module name as a string:

> np.lookfor('boundaries', 'skimage')

or the module itself:

> import skimage
> np.lookfor('boundaries', skimage)
```